### ISSUE TITLE
[Feature Request] Support One to Many user mapping Plex to Jellyfin

### ISSUE THREAD

**Issue by burneystarke** [luigi311/JellyPlex-Watched#127]
**Is your feature request related to a problem? Please describe.**
Using a usermapping like
USER_MAPPING = { "plexuser": "jellyfinuser1", "plexuser":"jellyfinuser2", "plexuser":"jellyfinuser3" }
results in only mapping plexuser to jellyfinuser3

**Describe the solution you'd like**
Support one to many mapping plex->jellyfin

**Describe alternatives you've considered**
Mapping Plex to Jellyfin for the primary user then Jellyfin to Jellyfin for the other users.

**Additional context**
I am not well versed in python, but this appears to be a due to:
- combine_user_lists function using a hashtable and overriding the mapped user as it iterates
- generate_server_connections function returning plex first 

I am not sure if it would be better to allow the `users[plexuser]` to be an array type, or for `users` object to be an array of hashes? 

I'll try the cascading sync with multiple copies of the same JellyFin server. Thank you for your time and effort in building this tool for us all.

**Comment by burneystarke** [comment]
multiple copies of the same Jellyfin server did not work for me. For now I'll run multiple copies of Jellyplex-watched

**Comment by luigi311** [comment]
Yeah this doesnt work in the current implementation due to how i have some of the functions working to support the mapping. I will need to redesign some of the functions so this is possible without breaking the current .env formatting. Until then multiple instances of jellyplex-watched is definitely the way to go
